dev/com.ibm.ws.anno_fat/fat/src/com/ibm/ws/tests/anno/caching/AnnotationScanInJarTest.java

```diff
@@ -0,0 +1,121 @@
+package com.ibm.ws.tests.anno.caching;
+
+import static org.junit.Assert.assertTrue;
+
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.EERepeatActions;
+import componenttest.rules.repeater.RepeatTests;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.FATServletClient;
+import spring.test.init.jar.JarInit;
+import spring.test.init.manifest.ManifestInit;
+import spring.test.init.sharedlib.SharedLibInit;
+import spring.test.init.war.WebInit;
+
+@RunWith(FATRunner.class)
+public class AnnotationScanInJarTest extends FATServletClient {
+
+	public static final String APP_NAME = "springTest";
+	public static final String SERVER_NAME = "springTest_server";
+
+	@Server(SERVER_NAME)
+	public static LibertyServer server;
+
+	@ClassRule
+	public static RepeatTests r = EERepeatActions.repeat(SERVER_NAME,
```

Comment:
```suggestion
	public static RepeatTests r = EERepeatActions.repeat(SERVER_NAME, true,
```

No need to transform if it is already an EE 10 application.